lib/views/components/async/logo_progress_indicator.dart

```diff
@@ -0,0 +1,18 @@
+import "package:flutter/material.dart";
+
+/// A progress indicator that displays the animated Proxima logo.
+/// Used to indicate that the app is loading data. And provide an
+/// alternative to the default circular progress indicator.
+class LogoProgressIndicator extends StatelessWidget {
+  const LogoProgressIndicator({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Image.asset(
+      "assets/feedback/proxima_progress_indicator_transparent.gif",
```

Comment:
Can you please place this in a private static const, similar to the app logo in `lib/views/pages/login/login_page.dart`

Reply:
You are completely right !

Fixed in: [e2f1e0b](https://github.com/ProximaEPFL/proxima/pull/315/commits/e2f1e0b85e804f5ddfd0ab2e532ccb0b79ce763d)

Thank you 💯  !